.github/workflows/images-cleanup.yaml

```diff
@@ -8,7 +8,7 @@ permissions:
   packages: write
 
 env:
-  BASE_REPOSITORY: warmmetal/csi-driver-image
+  BASE_REPOSITORY: warm-metal/csi-driver-image
```

Comment:
The name issue also affected another workflow - https://github.com/warm-metal/csi-driver-image/pull/113#issue-2080441482. Fixed it in this PR related to stale issues.

Reply:
Yeah that's a easy thing to trip on, I will add a note to catch such issues in future during reviews.